daras_ai_v2/gdrive_downloader.py

```diff
@@ -9,6 +9,8 @@
 def is_gdrive_url(f: furl) -> bool:
     return f.host in ["drive.google.com", "docs.google.com"]
 
+def is_gdrive_presentation_url(f: furl) -> bool:
+    return is_gdrive_url(f) and "presentation" in f.path.segments
```

Comment:
hmm, this looks wrong, you only want docs.google.com URLs here no?  drive.google.com implies a pptx uploaded to google drive which is different  from a docs.google.com google slides url

Reply:
Yes, only docs.google.com supports scrolling directly to the desired slide. I initially thought appending #slide=4 to the URL would provide meaningful context for users when sharing/reading a Google Drive preview link for a PowerPoint file.  example:
https://drive.google.com/file/d/1abcdef/view#slide=4 – but this doesn't scroll down to the slide 4